Doc/glossary.rst

```diff
@@ -1039,8 +1040,8 @@ Glossary
    type alias
       A synonym for a type, created by assigning the type to an identifier.
 
-      Type aliases are useful for simplifying complex
-      :term:`type hints <type hint>`.  For example ::
+      Type aliases are useful for simplifying :term:`type hints <type hint>`.
+      For example ::
```

Comment:
Honestly this example would look better if it read "For example [bad code] could be made more readable like this [good code]" (currently it shows [good code] first).